src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CspParameters.cs

```diff
@@ -48,6 +48,18 @@ public CspProviderFlags Flags
             }
         }
 
+        public SecureString KeyPassword
```

Comment:
```c#
public SecureString KeyPassword
{
    get { return null; }
    set { throw new PlatformNotSupportedException(); }
}
```
Feels a bit more gentle (and doesn't make the debugger angry that exceptions were thrown from a get).